doc/en/components/general-changelog-dv-react.md

```diff
@@ -9,6 +9,12 @@ namespace: Infragistics.Controls.Charts
 
 All notable changes for each version of {ProductName} are documented on this page.
 
+## **{PackageVerChanges-23-2}**
+
+### New Components
+
+* [Grid](grids/data-grid.md)
```

Comment:
Well I see that the patter is the same with Blazor where it's just a link to the topic but we can do better. Explain that this is a new, fully functional cross-platform grid, and add a deprecation section where you say that the old IgrDataGrid is deprecated.